src/assets/styles/style.less

```diff
@@ -32,19 +33,11 @@ body {
 .main {
   background-color: @dark-background;
   padding: 0px;
-  position: fixed;
-  top: 0px;
-  bottom: 0px;
-  left: 0px;
-  right: 0px;
+  .position(fixed, 0px, 0px, 0px, 0px);
```

Comment:
Thinking about it we should add the names. It's more readable and more resilient to the function changing in the future.